### PR TITLE
RHDEVDOCS-4436 Update OP 1.7 Release Note

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 4.9+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-4436
- Direct link to doc preview: tbd
- SME review: @vdemeester 
- QE review: @ppitonak 
- Peer review: @ 	tbd
- <All reviews complete. Please merge now.>

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: modules/op-release-notes-1-7.adoc [hunk 1]
```diff
@@ -252,7 +252,7 @@ Pipelines as Code supports the following features:
 == Deprecated features
 
 // Pipelines
-* Breaking change: The `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` custom resource (CR) is removed. As a result, the `$HOME` environment variable and `workingDir` parameter are unavailable.
+* Breaking change: This update removes the `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields from the `TektonConfig` custom resource (CR). As a result, the `TektonConfig` CR no longer automatically sets the `$HOME` environment variable and `workingDir` parameter. You can still set the `$HOME` environment variable and `workingDir` parameter by using the <something> and <something_else> fields in the  `Task` custom resource definition (CRD).
```
Comment (vdemeester) [inline modules/op-release-notes-1-7.adoc:255]: something = the `env` field
something_else: the `workingDir` field

e.g.

```
workingDir: /workspace/foo
env:
- name: HOME
  value: /tekton/home
```

**Comment by rolfedh** [comment]
/assign @ppitonak 
Please provide QE review and approval. Thanks!

**Review by ppitonak — APPROVED** [review]

**Pull request merged**